ehcache-V3-test/pom.xml

```diff
@@ -20,17 +20,17 @@
     <!-- Implementation coordinates -->
     <implementation-groupId>org.ehcache</implementation-groupId>
     <implementation-artifactId>ehcache</implementation-artifactId>
-    <implementation-version>3.0.0.m3</implementation-version>
+    <implementation-version>3.0.1</implementation-version>
 
     <!-- CacheManager and Cache implementation. Used by the unwrap tests. -->
-    <CacheManagerImpl>org.ehcache.jsr107.Eh107CacheManager</CacheManagerImpl>
-    <CacheImpl>org.ehcache.jsr107.Eh107Cache</CacheImpl>
-    <CacheEntryImpl></CacheEntryImpl>
+    <CacheManagerImpl>org.ehcache.CacheManager</CacheManagerImpl>
+    <CacheImpl>org.ehcache.Cache</CacheImpl>
+    <CacheEntryImpl>org.ehcache.Cache$Entry</CacheEntryImpl>
 
     <!--The MBeanServer, so that the TCK can resolve it. -->
-    <javax.management.builder.initial>org.ehcache.jsr107.Eh107MBeanServerBuilder
+    <javax.management.builder.initial>org.ehcache.jsr107.internal.tck.Eh107MBeanServerBuilder
     </javax.management.builder.initial>
-    <org.jsr107.tck.management.agentId>anyvalue</org.jsr107.tck.management.agentId>
+    <org.jsr107.tck.management.agentId>Eh107MBeanServer</org.jsr107.tck.management.agentId>
```

Comment:
Please try with `anyvalue`, it works unless you changed something. I use `anyvalue` to say that the value of the parameter isn't relevant, it just needs to be something.

Or do you have applications that use the Eh107MBeanServer agant id, too?

BTW, do you have an opinion on:
https://github.com/jsr107/jsr107tck/issues/83